deptry/utils.py

```diff
@@ -39,3 +41,64 @@ def load_pyproject_toml(pyproject_toml_path: str = PYPROJECT_TOML_PATH) -> Dict[
             return tomllib.load(pyproject_file)
     except FileNotFoundError:
         raise FileNotFoundError(f"No file `pyproject.toml` found in directory {os.getcwd()}")
+
+
+def find_site_packages_below(root: Path, platform: str) -> Optional[Path]:
```

Comment:
I would like to propose moving these functions outside of `utils.py`, and to a dedicated file + class.